simphony/cuds/lattice.py

```diff
@@ -25,30 +24,33 @@ def __init__(self, index, data=None):
             self.data = DataContainer(data)
 
 
-class Lattice(ABCLattice):
-    """
-    A Bravais lattice;
-    stores references to data containers (node related data).
+class MetaLattice(ABCLattice):
```

Comment:
BaseLattice is fine. We can use it for now, and if we can come up with something else in future then it's easy to change.